lib/gulp-tasks/patterns-import.js

```diff
@@ -0,0 +1,72 @@
+/**
+ *  @fileOverview
+ * Uses Gulpjs to import html patterns
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:gulp
+ *  @requires     NPM:lodash.merge
+ *  @requires     NPM:pattern-importer
+ */
+
+var gulp = require('gulp'),
+  merge = require('lodash.merge'),
+  utility = require('../utility');
+
+/**
+ * Function to get default options for an implementation of patternlab-import
+ * @returns {Object}  options  an object of default patternlab-import options
+ */
+var getDefaultOptions = function () {
```

Comment:
I believe the harm is in more confusing-to-read code. I also believe it's ok to do things the node way, like declare the `var whatever = require('')` statements outside functions, because even though it's javascript, it won't be used in a browser where those conventions are required.

And if any of this is ever used in a browser, the will be pulled in via something like browserify, which encapsulates each script anyway, so these issues still won't matter.